backend/src/handlers/user_handler.go

```diff
@@ -17,14 +17,16 @@ func (srv *Server) registerUserRoutes() []routeDef {
 	return []routeDef{
 		{"GET /api/users", srv.handleIndexUsers, true, axx},
 		{"GET /api/users/{id}", srv.handleShowUser, false, axx},
+		{"GET /api/users/{id}/program-history", srv.handleGetUserProgramHistory, false, axx},
+		{"GET /api/users/{id}/weekly-schedule", srv.handleGetUserWeeklySchedule, false, axx},
```

Comment:
@corypride Can we look at the handler in `class_events.go`? 

`GET /api/student-calendar", srv.handleGetStudentCalendar` should do what we need. But I think Preston is pushing some changes it needs to support a date range.